composer.json

```diff
@@ -29,7 +29,8 @@
         "phpunit/php-text-template": "^1.2",
         "sebastian/code-unit-reverse-lookup": "^1.0",
         "sebastian/environment": "^2.0",
-        "sebastian/version": "^2.0"
+        "sebastian/version": "^2.0",
+        "theseer/tokenizer": "^1.1.0"
```

Comment:
Lol. True. What I really meant was, this should probably changed to that, to match the rest of the file.